python3/help.sh

```diff
@@ -8,11 +8,21 @@ else
 
     echo "OVERVIEW: run python3 with SplashKit settings"
     echo
-    echo "USAGE: skm python3 [options] [input]"
+    echo "USAGE: python3 [options] [input]"
```

Comment:
this should still be the help for running it through skm. but maybe have this as a note that they should be able to run it without.